static/scss/profile.scss

```diff
@@ -58,16 +73,44 @@
     }
     .profile-form-row > .basic-info {
       width: 50%;
+      &.full-width {
+        width: 100%;
+        padding: 20px 0;
+      }
+      &.border-bottom {
+        border-bottom: 2px #E2E2E2 solid;
+        margin-right: 10px;
+        &.education-top-spacing {
+          margin-top: 17px;
+        }
+        &.employment-top-spacing {
+          margin-top: -11px;
+        }
+      }
       .school-type {
-        font-size: 12px;
-        font-weight: 400;
-        text-transform: lowercase;
+        margin-bottom: -10px;
+        font-size: 16px;
+        font-weight: 600;
+        text-transform: capitalize;
+      }
+      .profile-row-name {
+        font-size: 16px;
+        color: #404040;
+      }
+      .profile-row-date-range {
+        font-size: 16px;
+        font-weight: 300;
+        color: #b7b7b7;
+      }
+      .profile-row-new-line {
+        display: block;
+        width: 100%;
       }
     }
 
     .profile-card-header {
       > span {
-        font-size: 18px;
+        font-size: 22px;
```

Comment:
I think it should be 18px on the userpage anyway, that makes it look closer to the mockup